### ISSUE TITLE
docker info yields illegal base64 data at input byte 0 - why?  Important?

### ISSUE THREAD

**Issue by coderlol** [docker/docker#13285]
ubuntu@ip-10-2-1-11:~$ docker info
Containers: 0
Images: 0
Storage Driver: aufs
 Root Dir: /var/lib/docker/aufs
 Backing Filesystem: extfs
 Dirs: 0
 Dirperm1 Supported: false
Execution Driver: native-0.2
Kernel Version: 3.13.0-52-generic
Operating System: Ubuntu 14.04.2 LTS
CPUs: 2
Total Memory: 3.859 GiB
Name: ip-10-2-1-11
ID: YE47:FSWB:BQ6C:XWIG:UAMF:AUTH:7G6W:UF7K:G5DV:3ZPK:S7LS:NQVY
WARNING: illegal base64 data at input byte 0
WARNING: No swap limit support
ubuntu@ip-10-2-1-11:~$ docker --version
Docker version 1.6.2, build 7c8fca2
ubuntu@ip-10-2-1-11:~$ echo $TERM
cygwin
ubuntu@ip-10-2-1-11:~$

**Comment by runcom** [comment]
uhm seems like something with cliConfig when decoding auth maybe, ping @duglin

**Comment by duglin** [comment]
@coderlol couple of things...
1 - what does your ~/.dockercfg file look like?  Don't paste it into the issue because it contains passwords, but can you describe what it looks like? Or mask-out the "auth" field and then paste it.  See if there are any weird hex values in there
2 - can you try on the latest Docker?

I tried to reproduce this on docker 1.6.2 but wasn't able to.

**Comment by coderlol** [comment]
{"https://blablabla.com/v2/":{"auth":"$apr1$7Jt8M8WS$sJeq7WppW9SpykWKTLcYN1","email":"ughugh@blablabla.com"}}

I used docker 1.6.2.  There is a more recent version?  I have noticed that after a few docker ops such as docker run docker build, the issue seems to go away.

I do see the issue on a "pristine" EC2 instance.  Started the instance, tried docker pull and got the issue.  Furthermore, docker pull insisted that I provide a login although there is a .dockercfg.  Perhaps, the .dockercfg is corrupted as you are suspecting.
